docs/source/examples/mg-mode-part-ii.ipynb

```diff
@@ -0,0 +1,1684 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Instantiate some Nuclides\n",
+    "h1 = openmc.Nuclide('H1')\n",
+    "b10 = openmc.Nuclide('B10')\n",
+    "o16 = openmc.Nuclide('O16')\n",
+    "u235 = openmc.Nuclide('U235')\n",
+    "u238 = openmc.Nuclide('U238')\n",
+    "zr90 = openmc.Nuclide('Zr90')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the nuclides we defined, we will now create three materials for the fuel, water, and cladding of the fuel pins."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# 1.6% enriched fuel\n",
+    "fuel = openmc.Material(name='1.6% Fuel')\n",
+    "fuel.set_density('g/cm3', 10.31341)\n",
+    "fuel.add_nuclide(u235, 3.7503e-4)\n",
+    "fuel.add_nuclide(u238, 2.2625e-2)\n",
+    "fuel.add_nuclide(o16, 4.6007e-2)\n",
+    "\n",
+    "# zircaloy\n",
+    "zircaloy = openmc.Material(name='Zircaloy')\n",
+    "zircaloy.set_density('g/cm3', 6.55)\n",
+    "zircaloy.add_nuclide(zr90, 7.2758e-3)\n",
+    "\n",
+    "# borated water\n",
+    "water = openmc.Material(name='Borated Water')\n",
+    "water.set_density('g/cm3', 0.740582)\n",
+    "water.add_nuclide(h1, 4.9457e-2)\n",
+    "water.add_nuclide(o16, 2.4732e-2)\n",
+    "water.add_nuclide(b10, 8.0042e-6)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With our three materials, we can now create a Materials object that can be exported to an actual XML file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Instantiate a Materials object\n",
+    "materials_file = openmc.Materials((fuel, zircaloy, water))\n",
+    "\n",
+    "# Export to \"materials.xml\"\n",
+    "materials_file.export_to_xml()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's move on to the geometry. This problem will be a square array of fuel pins and control rod guide tubes for which we can use OpenMC's lattice/universe feature. The basic universe will have three regions for the fuel, the clad, and the surrounding coolant. The first step is to create the bounding surfaces for fuel and clad, as well as the outer bounding surfaces of the problem."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Create cylinders for the fuel and clad\n",
+    "fuel_outer_radius = openmc.ZCylinder(x0=0.0, y0=0.0, R=0.39218)\n",
+    "clad_outer_radius = openmc.ZCylinder(x0=0.0, y0=0.0, R=0.45720)\n",
```

Comment:
I note these are the default values now, so the users know going forward.

- [x]